src/OpenDB/src/swig/common/swig_common.cpp

```diff
@@ -123,18 +123,21 @@ int write_def(odb::dbBlock* block,
 
 int write_lef(odb::dbLib* lib, const char* path)
 {
-  odb::lefout writer;
+  utl::Logger* logger = new utl::Logger(NULL);
```

Comment:
Why wouldn't you get the OR logger instead of creating a new one?

Reply:
It wasn't clear to me that the OR logger was accessible in this scope. Other methods had been doing this. @maliberty care to weigh in?

Reply:
It looks like this is for the standalone OpenDB tcl/python interfaces. The actual openROAD app only uses a single logger. I'm guessing in the independent OpenDB binaries wouldn't have access to a common logger is that correct?

Reply:
There are no standalone opendb binaries anymore.

Reply:
I feel this is not right but as the problem is preexisting and more widespread we can tackle it in another general PR.